haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Compat.hs

```diff
@@ -420,6 +420,27 @@ pattern IEVarCompat lwn <-
   IEVar _ lwn
 #endif
 
+pattern IEThingAbsCompat lwn <-
+#if __GLASGOW_HASKELL__ < 806
```

Comment:
For what it's worth, I'd support removing everything lower than 8.6 given how significantly the API changed in that version.

Reply:
I filed https://github.com/google/haskell-indexer/issues/98, which is about removing support for 8.0. But yeah, I want to remove all the versions <0.6.

But until that happens, I think we should keep the builds green. Hopefully I can work on that soon once we have a consensus.

Reply:
All right. I changed the title of #98 so repurpose it to removing versions <8.6 and supporting 8.8.